routing/scheme.rst

```diff
@@ -94,3 +94,10 @@ to always use ``http``.
     to secure an "area" of your website (all URLs under ``/admin``) or when
     you want to secure URLs defined in a third party bundle (see
     :doc:`/security/force_https` for more details).
+
+.. note::
+
+    Inside CLI commands or the messenger workers the router context is not defined
```

Comment:
no need to write messenger workers, are they are cli commands, no?

Reply:
I added messenger workers because they are not Symfony commands. But the common trait - it is running in CLI mode. How about this version?
`When a script is running in CLI mode (e.g Symfony commands or messenger workers) the router context is not defined`